python/ecl/grid/ecl_grid.py

```diff
@@ -1262,6 +1262,16 @@ def create_volume_keyword(self, active_size=True):
         return self._create_volume_keyword(active_size)
 
     def export_index(self, active_only = False):
+        """
+        Exports a pandas dataframe, index_frame, containing index data of grid cells.
+
+        index_frame.index contains global_index for each cell listed.
+        column3 0, 1, 2 are i, j, k, respectively
+        column 3 contains the active_index
+        if active_only == True, only active cells are listed, 
+        otherwise all cells are listed.
+
```

Comment:
This index frame should typically be passed to the `epxport_data(), export_volume()` and `export_corners()` functions.